src/file.cpp

```diff
@@ -80,18 +80,18 @@ namespace Sass {
     bool file_exists(const std::string& path)
     {
       #ifdef _WIN32
-        wchar_t resolved[32768];
+        std::vector<wchar_t> resolved(32768);
```

Comment:
Why is this so big anyway?
Should use `MAX_PATH` instead on Windows, which is only 260

https://docs.microsoft.com/en-us/windows/win32/fileio/naming-a-file#maximum-path-length-limitation

Reply:
Alternative is to enable long paths: https://docs.microsoft.com/en-us/windows/win32/fileio/naming-a-file#enable-long-paths-in-windows-10-version-1607-and-later

Reply:
> To extend this limit to 32,767 wide characters, call the Unicode version of the function (GetFullPathNameW), and prepend "\?" to the path.

We do use GetFullPathNameW and already use long UNC paths.